docs/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add `MockOracle.sol` for testing contracts
+- Added a new boolean `OPTIMISM_GAS_FEES` configuration variable which makes a call to estimate gas before all transactions, suitable for use with Optimism's L2 chain. When this option is used `ETH_GAS_LIMIT_DEFAULT` is ignored.
```

Comment:
Should we log an INFO message on app boot if this is enabled?

How about bombing out with an error if the ChainID doesn't match Optimism? Or even autosetting it from the chain ID?

Reply:
Bit hesitant to add just yet, since it'll break if people try to use with chain ID's we don't recognize and it's pretty obviously only applicable to Optimism.